adb_graphics/figure_builders.py

```diff
@@ -117,6 +121,27 @@ def parallel_maps(cla, fhr, ds, level, model, spec, variable, workdir,
             print(f'Cannot find grib2 variable for {variable} at {level}. Skipping.')
             return
 
+        if cla.graphic_type == "diff":
+
+            field2 = gribdata.fieldData(
+                ds=ds2,
+                fhr=fhr,
+                filetype=cla.file_type,
+                level=level,
+                member=mem,
+                model=model,
+                short_name=variable,
+                )
+
+            try:
+                field2.field
+            except errors.GribReadError:
+                print((f'Cannot find grib2 variable for {variable} at {level} in',
+                       '2nd set of files. Skipping.'))
+                return
+
+            field.values = field.values() - field2.values()
```

Comment:
I actually need to fix this. We will need a setter/getter to make this work appropriately!